bibtex/readme.md

```diff
@@ -0,0 +1,19 @@
+# BibTeX Grammar
+BibTeX grammar for ANTLR v4.
```

Comment:
Please use line break between header and paragraph in markdown files:

```suggestion
# BibTeX Grammar

BibTeX grammar for ANTLR v4.
```

Reply:
Thanks for the suggestion. The readme file has been formatted.